.github/workflows/test_dbt_fal.yml

```diff
@@ -0,0 +1,167 @@
+name: Test dbt-fal adapter
+
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
```

Comment:
```suggestion
    types: [assigned, opened, synchronize, reopened]
    branches: ['act1**']
```

Reply:
I thought it would do that by default  (e.g. only work on branches that inherit from `act1`)

Reply:
you mean because of the `push` instruction? that is isolated from this one

Reply:
Well, I guess that is true.